### ISSUE TITLE
VideoClicks instead of VideoClips

### ISSUE THREAD

**Issue by schingeldi** [sokil/php-vast#1]
There is no VideoClips node in VAST. Shouldn't it be VideoClicks in \Ad\InLine\Creative\Linear.php

I think this function is wrong:

```
private function _getVideoClipsDomElement()
    {
        // create container
        if($this->_videoClicksDomElement) {
            return $this->_videoClicksDomElement;
        }

        $this->_videoClicksDomElement = $this->_domElement->getElementsByTagName('VideoClips')->item(0);
        if($this->_videoClicksDomElement) {
            return $this->_videoClicksDomElement;
        }

        $this->_videoClicksDomElement = $this->_domElement->ownerDocument->createElement('VideoClips');
        $this->_domElement->firstChild->appendChild($this->_videoClicksDomElement);

        return $this->_videoClicksDomElement;
    }
```

**Comment by sokil** [comment]
Yes, you are right. It must be `2.3.1.6 The Optional <VideoClicks> Element`. I'l fix this soon. Thank you

**Comment by sokil** [comment]
Fixed in 0.2.1
